### DISCUSSION TITLE
Is it possible to port SU2 code to GPU using CUDA C++ ?

### DISCUSSION THREAD

**Discussion by psgmech** [su2code/SU2#1409] [Q&A]
I found parallel execution commands for SU2 test case using mpirun. For example, to run the CFD solver on 8 cores, you might enter: $ mpirun -n 8 SU2_CFD default.cfg

Also noted OpenMP header files and c++ files at ~SU2/Common/include/parallelization/ path.

OpenMP (shared memory) – Parallel programming on a single node.
MPI (distributed memory) – Parallel computing running on multiple nodes.

SU2 supports both OpenMP and MPI. So, we hope that CUDA C++ can be supported by SU2. Is it possible to port SU2 code to GPU using CUDA C++ ?

**Comment by zazabap** [comment]
Probably this feature is getting more and more attractive. However, currently seems no one is working on this implementation.

**Reply by pcarruscag** [reply]
https://github.com/su2code/SU2/pull/2346
